model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -163,10 +182,15 @@ def nodes_builder(model: GraphModule,
                 framework_attr_filtered[k] = v
         framework_attr = framework_attr_filtered
 
-        # filter Nodes from node kwargs, we replace these attributes with nx graph nodes
+        # filter Nodes from node kwargs, we replace these attributes with nx graph nodes. The Node kwargs
```

Comment:
maybe try to clarify this as well, I'm afraid it is written in a way that only you can understand because you have the explanation in your head.
I think that there is no escape from providing an example.